### PR TITLE
Separate CI pipelines for communication service directory.

### PR DESCRIPTION
**Pull request by pallavit**

### CODE REVIEW

**Comment by pallavit** [comment]
@hallipr I created a PR to separate communication pipelines as we had discussed yesterday. Can you please review and let me know what is the best way to cleanup older pipelines and test newer ones :)

File: sdk/communication/azure-communication-chat/ci.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      - hotfix/*
+      - release/*
+  paths:
+    include:
+      - sdk/communication/azure-communication-chat/
+    exclude:
+      - sdk/communication/azure-communication-chat/pom.xml
+
+extends:
+  template: ../../eng/pipelines/templates/stages/archetype-sdk-client.yml
```
Comment (weshaggard) [inline sdk/communication/azure-communication-chat/ci.yml:29]:
```suggestion
  template: /eng/pipelines/templates/stages/archetype-sdk-client.yml
```
Similar for the others

Comment (pallavit) [reply sdk/communication/azure-communication-chat/ci.yml:29]: Isn't this path relative to this file? If so how does '/eng/pipelines' work here?

**Comment by pallavit** [comment]
> @pallavit please also make sure the dependencies between these libraires are using the "dependency" version type instead of "current".

Yes, they are already using dependency tag?

File: sdk/communication/azure-communication-chat/ci.yml (hunk 1 above)
Comment (pallavit) [reply sdk/communication/azure-communication-chat/ci.yml:29]: I got my answer. It is relative to the root! Updating ...

Comment (weshaggard) [reply sdk/communication/azure-communication-chat/ci.yml]: Yes starting with / makes it absolute from the root.

**Review by benbp — APPROVED** [review]

**Comment by pallavit** [comment]
Trying to create a common ci template got way more complex, so I am letting this stay as-is.

**Pull request merged**